src/status_im/ui/screens/browser/default_dapps.cljs

```diff
@@ -16,6 +16,10 @@
              :dapp-url    "https://app.ercdex.com/"
              :photo-path  "contacts://erc-dex"
              :description "Trustless trading has arrived on Ethereum"}
+             {:name        "dBay"
+             :dapp-url    "https://www.dbay.ai/"
+             :photo-path  "contacts://dBay"
+             :description "Buy all Yours dApps in one place."}
```

Comment:
Suggested copy tweak: `Buy from all your favorite DApps in one place.` You guys trade ERC721s, right, @LuisEGV?

Reply:
I just changed this commit to fit the description you suggested and also included dBay in the marketplace section. Thanks for that Rachel :)

Reply:
Thanks for the changes @LuisEGV!